<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>397</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string> Base of olympiads</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QPushButton{
	height: 60px;
	font-family: 'Montserrat', sans-serif;
	font-size: 11px;
	text-transform: uppercase;
	font-weight: 600;
	color: rgb(205, 92, 92);
	background-color: rgba(255, 255, 255, 0);
	border: none;
}

QPushButton:hover{
	height: 75px;
	border: 1px solid rgba(205, 92, 92, 1) ;
	border-radius: 5px;
	font-family: 'Montserrat', sans-serif;
	font-size: 11px;
	text-transform: uppercase;
	font-weight: 600;
	color: rgb(205, 92, 92);
	background-color: rgba(255, 255, 255, 0);
}

QPushButton:pressed{
	background-color: rgba(205, 92, 92, 1);
	color: white;
	border:2px solid;
	border: 1px solid rgba(205, 92, 92, 1) ;
	border-radius: 5px;
	font-family: 'Montserrat', sans-serif;

}

QLabel{
	font-family: 'Montserrat', sans-serif;
	color: rgb(71, 74, 81);
}
</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="rightMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <family>Montserrat</family>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Olympiad</string>
          </property>
          <property name="textFormat">
           <enum>Qt::AutoText</enum>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="but_olymp_add">
          <property name="text">
           <string>Add</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <family>Montserrat</family>
            <pointsize>20</pointsize>
            <kerning>true</kerning>
           </font>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string>Game</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="but_game_add">
          <property name="text">
           <string>add</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="but_del">
      <property name="text">
       <string>del</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="but_view">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>view data</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>397</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
